test/runtests.jl

```diff
@@ -504,6 +512,34 @@ end
         4.815559090394707, 2.3612485765861515, 3.8867779526777784]
 end
 
+@testset "GLM confidence intervals" begin
+    X = [fill(1,50) range(0,1, length=50)]
+    Y = vec([0 0 0 1 0 1 1 0 0 0 0 0 0 0 1 0 1 1 0 1 1 0 1 0 0 1 1 1 0 1 1 1 1 1 0 1 0 1 1 1 0 1 1 1 1 1 1 1 1 1])
+    gm = fit(GeneralizedLinearModel, X, Y, Binomial())
+
+    newX = [fill(1,5) [0.0000000, 0.2405063, 0.4936709, 0.7468354, 1.0000000]]
+
+    ggplot_prediction = [0.1804678, 0.3717731, 0.6262062, 0.8258605, 0.9306787]
+    ggplot_lower = [0.05704968, 0.20624382, 0.46235427, 0.63065189, 0.73579237]
+    ggplot_upper = [0.4449066, 0.5740713, 0.7654544, 0.9294403, 0.9847846]
+
+    R_glm_se = [0.09748766, 0.09808412, 0.07963897, 0.07495792, 0.05177654]
+
+    preds_asymmetric = predict(gm, newX, interval=:confidence, symmetric=false)
+    preds_symmetric = predict(gm, newX, interval=:confidence, symmetric=true)
+
+    @test preds_asymmetric.prediction == preds_symmetric.prediction
+    @test preds_asymmetric.prediction ≈ ggplot_prediction atol=1e-3
+    @test preds_asymmetric.lower ≈ ggplot_lower atol=1e-3
+    @test preds_asymmetric.upper ≈ ggplot_upper atol=1e-3
+
+    @test preds_symmetric.upper .-  preds_symmetric.lower ≈ 2 .* 1.96 .* R_glm_se atol=1e-3
+
+    # now fit model at level so that CI width is equal to standard error
```

Comment:
That sounds like a weird test IMHO. It's really the same as checking `2 .* 1.96 .* R_glm_se` as above.

Reply:
Fully agree, going to remove this!